### ISSUE TITLE
Support links without hostname in content editor

### ISSUE THREAD

**Issue by dominic** [eastbaydsa/website#80]
@wesholing was unable to add links to items without including `http://eastbaydsa-website-staging.herokuapp.com` at the front of the URL

**Comment by wesholing** [comment]
Yes! I tried it out last night, and it works. I'm in the process of going through and updating the rest of the pages I've flagged that had absolute URL links.
